python_dependencies/jupyter-pip-requirements.txt

```diff
@@ -1,3 +0,0 @@
-ipython==8.10.0
-notebook==6.5.6
-ipywidgets==7.7.1
```

Comment:
These are always required by the narrative, so I just added them to requirements.txt. We might need to fuss with dependabot to keep notebook frozen at 6.x.